charts/kueue/values.yaml

```diff
@@ -92,6 +92,11 @@ managerConfig:
     #    backoffBaseSeconds: 60
     #    backoffMaxSeconds: 3600
     #manageJobsWithoutQueueName: true
+    #managedJobsNamespaceSelector:
```

Comment:
Putting the comment here because the other place is not changed, but I would prefer to have the same selector commented out in `config/components/manager/controller_manager_config.yaml` for consistency
(it could be a dedicated PR we cherry-pick even). 
 I think having different comments in charts and configs will be confusing.
WDYT? 

I admit using the `kueue-namespace` label approach might be better and more flexible for users, but I would suggest documenting it better.

Reply:
I'll make them both be the same and also be the same as the "hidden" defaulting behavior.

Will prepare a separate PR just for that now.

Reply:
Done: https://github.com/kubernetes-sigs/kueue/pull/4262